grammar.js

```diff
@@ -121,11 +121,22 @@ module.exports = grammar({
       repeat($._blank),
     ),
 
+    // Language annotation for code blocks: only support core languages
+    language: ($) => choice(
+      'lua', 'vim', 'c'
```

Comment:
Surely it shouldn't be that much extra effort to allow any word here.

Reply:
Not at all; I deliberately wanted to be conservative for now (and string literals _are_ easier to get working for a noob like me).

Reply:
(Actually, it is more effort, as `$.word` doesn't work here.)

Reply:
But we could make a simple `token.immediate(/[a-z]*/)` regex, of course. Not sure that's better, though.

Reply:
Does it work?

Reply:
Sure.

Reply:
If it works then seems like the best route to me.

Reply:
Already done :)